### PR TITLE
Feeling Room Fine ; Animations

### LINKED ISSUE #38: Feel Room Visuals: Fine
- [ ] The feeling room should have a dynamic background unique to the feeling.
- [ ] Animations that loop once the user starts the experience.
- [ ] A color pallet of green.

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
#### Description
Adds animations to the `FeelingRoom` component for when the mood `Fine` is selected.
#### Motivation and Context
We want animations on all of the feeling rooms that reflect the unique emotion selected. 
#### How Has This Been Tested?
- [ ] In console
- [x] In browser
- [ ] With testing files
- [ ] Other Please specify:
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Styling / Design
- [ ] Testing
- [ ] Refactor
#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
#### Issues Addresses: 
Resolves #38

### CODE REVIEW

File: src/Components/FeelingRoom/fineAnimations.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import gsap from 'gsap'
+
```
Comment (Kristiannmiller) [inline src/Components/FeelingRoom/fineAnimations.js:2]: This animation looks great! I'd love to address the "blip" in the gradient if we have time, but it's definitely not a priority 1.

File: src/Components/FeelingRoom/fineAnimations.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
+
+@mixin circle($size, $bc, $z) {
```
Comment (Kristiannmiller) [inline src/Components/FeelingRoom/fineAnimations.scss:2]: This mixin is GREAT! Let's continue to use this as we do animations to keep our scss files a little more structured.

**Review by Kristiannmiller — APPROVED** [review]
What a great start to our Feeling Room animations! I love the floating "bubbles". Let's take a look at getting them to be enclosed in their parent element (I think whatever we come up with will also be helpful as we work through our divide and conquer work for the other rooms as well).

File: src/Components/FeelingRoom/FeelingRoom.js [hunk 3]
```diff
@@ -10,10 +12,35 @@ class FeelingRoom extends Component {
     }
   }
   createFineRoom = () => {
-    //functions to animate go here
+    fineBackgroundAnimation()
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/FeelingRoom.js:15]: I wonder if we could refactor this to a for loop or something a bit more compact?
Not an issue, but certainly something to consider for the future.

File: src/Components/FeelingRoom/FeelingRoom.js [hunk 4]
```diff
@@ -10,10 +12,35 @@ class FeelingRoom extends Component {
...
+    floatAnimation('#fineCircle9', 1)
+    floatAnimation('#fineCircle10', 1)
+    floatAnimation('#fineCircle11', 1)
+    floatAnimation('#fineCircle12', 1)
     return (
-      console.log('Fine')
-      //elements to animate go here
+      // console.log('Fine')
+      <section className='fineRoom'>
+        <div className='fineCircles' id='fineCircle1'></div>
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/FeelingRoom.js:31]: Also could be a use case for iteration.

File: src/Components/FeelingRoom/anxiousAnimations.scss [hunk 5]
```diff
@@ -0,0 +1,49 @@
+.anxiousCircle {
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/anxiousAnimations.scss]: This doesn't seem to be involved with the responsibility of this branch.

**Review by nathanielmillard — CHANGES_REQUESTED** [review]
Looking good, minus an addition file.
Can merge once thats gone :)

**Review by nathanielmillard — APPROVED** [review]
Glad to see that's resolved!
And excited to see our fine animation working well!

**Pull request merged**